components/salesforce/sources/new-object/new-object.js

```diff
@@ -2,26 +2,31 @@ const startCase = require("lodash/startCase");
 
 const common = require("../../common");
 
-const EVENT_SOURCE_NAME = "New Object (Instant)";
-
 module.exports = {
   ...common,
-  name: EVENT_SOURCE_NAME,
+  name: "New Object",
   key: "salesforce-new-object",
   description: "Emit an event when an object is created",
```

Comment:
Re: the above comment, maybe add something here re: the fact that this source lets you listen for new objects of any type, and that you choose the type when creating the source.

Reply:
Will do. How long are these descriptions supposed to be? Do we have a limit on their length?

Reply:
OK, I tried with a multiline string and it's handled fine by the UI so I'll go with that!